testing/data-scripts/sprint-14/TENANT_TEST_DATA.sql

```diff
@@ -1,35 +1,144 @@
-
-/* INSERTS */
-INSERT INTO [PIMS_TST].[dbo].[PIMS_LEASE_TENANT]
-    (LEASE_ID, ORGANIZATION_ID, LESSOR_TYPE_CODE, CONCURRENCY_CONTROL_NUMBER)
+-- Populate lease tenant data
+INSERT INTO
+    [dbo].[PIMS_LEASE_TENANT]
+    (
+    LEASE_ID,
+    ORGANIZATION_ID,
+    LESSOR_TYPE_CODE,
+    CONCURRENCY_CONTROL_NUMBER
+    )
 VALUES
-    (45, 48529, 'ORG', 1)
-INSERT INTO PIMS_CONTACT_METHOD
-    (CONCURRENCY_CONTROL_NUMBER, CONTACT_METHOD_TYPE_CODE, PERSON_ID, CONTACT_METHOD_VALUE)
+    (45, 2, 'ORG', 1)
+
+-- Insert contact methods
+INSERT INTO
+    PIMS_CONTACT_METHOD
+    (
+    CONCURRENCY_CONTROL_NUMBER,
+    CONTACT_METHOD_TYPE_CODE,
+    PERSON_ID,
+    CONTACT_METHOD_VALUE,
+    APP_CREATE_USERID,
+    APP_CREATE_USER_DIRECTORY,
+    APP_LAST_UPDATE_USERID,
+    APP_LAST_UPDATE_USER_DIRECTORY
+    )
 VALUES
-    (1, 'WORKMOBIL', 12566, '1111111111')
-INSERT INTO PIMS_CONTACT_METHOD
-    (CONCURRENCY_CONTROL_NUMBER, CONTACT_METHOD_TYPE_CODE, PERSON_ID, CONTACT_METHOD_VALUE)
+    (
+        1,
+        'WORKMOBIL',
+        1,
+        '1111111111',
+        'Test Data',
+        'Test Data',
+        'Test Data',
+        'Test Data'
+    )
+INSERT INTO
```

Comment:
not a big deal, but since the insert columns are the same, could add both rows in one insert instead of two for contact method, organization.

Reply:
+1 , It would be more valuable to have more 'realistic' test data. Otherwise, we risk having our synthetic data sets not reflect actual application ussage.

Reply:
@jdtoombs please log tech debt for the above, but don't do that for this PR. The purpose of this PR is just to get our test data working with our existing seed data, not to clean up the test data itself.

Reply:
Makes sense, will log!